### DISCUSSION TITLE
Red border line around pages

### DISCUSSION THREAD

**Discussion by TechNerdInfinity** [xournalpp/xournalpp#3449] [Q&A]
Hi,
Whenever I use xournal ++ there is a red line on the border of the pages, how can I remove this?
Thanks

**Comment by LittleHuba** [comment]
This red border Highlights the currently "selected" page. This is relevant for pasting content and other keyboard shortcuts that interact with a page. There is no way to deactivate this.

**Reply by personalizedrefrigerator** [reply]
The color of the border, however, can be changed in settings (I think under View...).

**Edit:** See below.

**Reply by arynvi** [reply]
> This red border Highlights the currently "selected" page. This is relevant for pasting content and other keyboard shortcuts that interact with a page. There is no way to deactivate this.

When you are in full page (sort of "presentation mode") there is no way you can mistake which is the currently selected page. 

In this case, the border is a nuisance since it prevents a proper full page presentation.

At least, this is my feeling while I give my presentations.

It would be nice, IMHO, to have an options which allows the user to disable the border. And, maybe, also an option which allows to choose its thickness, the same way you can choose the color.

**Comment by personalizedrefrigerator** [comment]
While the border can't be removed, its color can be changed.

To do this:
 1. Open settings (`Edit > Preferences`)
 2. Click `View`
 3. Change the `page border and selection color`
 
![](https://user-images.githubusercontent.com/46334387/137178345-e4f925f0-e914-49c1-8ebf-a0064215184f.png)
